lib/mpl_toolkits/axes_grid1/axes_grid.py

```diff
@@ -37,11 +37,17 @@ def toggle_label(self, b):
         axis = self.axis[self.orientation]
         axis.toggle(ticklabels=b, label=b)
 
-    def cla(self):
+    def clear(self):
+        # docstring inherited
+
         orientation = self.orientation
-        super().cla()
+        super().clear()
         self.orientation = orientation
 
+    def cla(self):
+        """Clear the current axes."""
+        self.clear()
```

Comment:
I think this instance of `cla()` does need to be kept around because this class isn't inheriting from Axes.